backend/pick-git/src/test/java/com/woowacourse/pickgit/unit/user/domain/UserRepositoryTest.java

```diff
@@ -290,12 +287,9 @@ void deleteUser_RelatedPostsRemoved_True() {
                 // given
                 User user = userRepository.findByBasicProfile_Name("testUser")
                     .orElseThrow(IllegalArgumentException::new);
-                Post post =
-                    new Post(new Images(new ArrayList<>()), new PostContent("hi"), "url", user);
-                Post post1 =
-                    new Post(new Images(new ArrayList<>()), new PostContent("hi"), "url", user);
-                Post post2 =
-                    new Post(new Images(new ArrayList<>()), new PostContent("hi"), "url", user);
+                Post post = Post.builder().author(user).build();
+                Post post1 = Post.builder().author(user).build();
+                Post post2 = Post.builder().author(user).build();
```

Comment:
```suggestion
                Post post = Post.builder()
                    .author(user)
                    .build();
                Post post1 = Post.builder()
                    .author(user)
                    .build();
                Post post2 = Post.builder()
                    .author(user)
                    .build();
```

이렇게 개행하면 읽기 더 편할 거 같아요!

Reply:
그리고 post 변수는 1부터 시작하는 게 어떤가요?

Reply:
테스트 코드는 이번 PR에서 리팩토링 대상이 아니기 때문에 추후 리팩토링을 생각하여 크게 신경쓰지 않았습니다.
이후 리팩토링에서 소스가 크게 바뀔 예정이니 그때 적용하도록 하겠습니다.